### ISSUE TITLE
Access Swagger API

### ISSUE THREAD

**Issue by hchev001** [hchev001/UniPS#53]
How does one go about accessing the swagger ui that you showed us the other day at school?

**Comment by fredlo2010** [comment]
I will work on it as soon as I finish the tests today. :)

**Comment by hchev001** [comment]
Awesome I finally found a video that explains how to talk with the back
end, very clear. So I started creating front end models to reflect the
backend models, but that may have to be refactored depending on what the
api calls will be returning. But for that I need swagger

On Mar 23, 2017 8:40 AM, "Alfredo" <notifications@github.com> wrote:

I will work on it as soon as I finish the tests today. :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/hchev001/UniPS/issues/53#issuecomment-288706149>, or mute
the thread
